apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -1711,6 +1711,29 @@ defmodule Astarte.RealmManagement.Queries do
     end
   end
 
+  def does_table_exist?(realm_name, table_name) do
```

Comment:
The convention in Elixir is not using the auxiliary verb when using `?`, so this should be `table_exists?`.

Moreover imho the `do_table_exists?` is small enough (and the resulting name is ugly enough) that you can just put the implementation inside the lambda here instead of having a separate function

Reply:
Done